tests/lib/rss.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * RSS Library Test
+ */
+
+const fs = require('fs')
+const rss = require('../../lib/rss')
+
+describe('lib.rss', () => {
+  let rssResponse;
+
+  beforeAll(async () => {
+    rssResponse = await rss({
+      data: JSON.parse(
+        fs.readFileSync(`${__dirname}/../__data__/data.json`)
+      ),
+      data_old: JSON.parse(
+        fs.readFileSync(`${__dirname}/../__data__/data_old.json`)
+      ),
+      feed_items: JSON.parse(
+        fs.readFileSync(`${__dirname}/../__data__/feed_items.json`)
+      ),
+      data_updated: new Date()
+    })
+  });
+
+  it('should create a feed.xml file', () => {
```

Comment:
But it's not creating any file right?

it's returning an object containing the feed?